es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchDAOV5.java

```diff
@@ -135,21 +135,11 @@
     }
 
     @Inject
-    public ElasticSearchDAOV5(Client elasticSearchClient, Configuration config, ObjectMapper objectMapper) {
+    public ElasticSearchDAOV5(Client elasticSearchClient, ElasticSearchConfiguration config, ObjectMapper objectMapper) {
         this.objectMapper = objectMapper;
         this.elasticSearchClient = elasticSearchClient;
-        this.indexName = config.getProperty("workflow.elasticsearch.index.name", null);
-        this.logIndexPrefix = config.getProperty("workflow.elasticsearch.tasklog.index.name", "task_log");
-
-        try {
```

Comment:
The section for `initIndex`, `updateIndexName` is happening now during the `setup`. 
The code is already there. This section got in here due to a bad resolution when resolving merge conflicts for https://github.com/Netflix/conductor/pull/669